content/cloud/labels.md

```diff
@@ -0,0 +1,65 @@
+title = "Labels"
```

Comment:
I think we've talked before about how the feature should be called "links" where the string name within the link is a label. For example, Apps and resources are linked together using a label, but if you think presenting it this way is clearer, I'm fine with that.